docs/content/preview/yugabyte-platform/upgrade/_index.md

```diff
@@ -33,6 +33,12 @@ If you are running YBA on a [deprecated OS](../../reference/configuration/operat
 
 {{< /note >}}
 
+{{< note title="Upgrading YBA on systems that have older versions of Python" >}}
+
+YBA v2024.1 and later requires Python v3.10-3.12. If you are running YBA on a system with Python earlier than 3.10, you will need to update Python on your system before you can upgrade YBA to v2024.1 or later.
```

Comment:
2025.1

Reply:
Or actually 2.25+, bc that will be the preview before 2025.1 released. we don't even have that on docs yet though so maybe this isn't needed?